### DISCUSSION TITLE
PatchTool 6.1 4/13

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#128] [Announcements]
僅做6.1版本相容性更新，可進行遊戲，尚未對新內容進行漢化。

同時修正了 #122

- FFXIVChnTextPatch-0413.rar：漢化器，一般來說只需下載這個。backup資料夾內含乾淨的6.1檔案，有需要可自取。
- rawexd-zhcn-0413.rar：簡體中文CSV，有需要者更名為rawexd後替換。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.58'>PatchTool 6.1 4/13</a>.</em>

**Comment by earvinwu** [comment]
感謝大大

**Comment by sai750310** [comment]
辛苦了

**Comment by lincTHU** [comment]
用国服6.08包体汉化后
![image](https://user-images.githubusercontent.com/8018928/163101595-3a6f3878-e62e-41fc-9add-940b5d855d5a.png)
用简体CSV汉化
![image](https://user-images.githubusercontent.com/8018928/163101962-b94d9097-95ff-4f5b-9e30-5cfa7cc81033.png)
应该是同一个问题

**Reply by lincTHU** [reply]
补充：繁体可正常进入游戏

**Reply by lincTHU** [reply]
补充：
我发现FFXIVChnTextPatch-0407版本+国服简体包汉化成功了。。。有点迷
有兴趣的可以试下。
![image](https://user-images.githubusercontent.com/8018928/163108436-261f3b52-b205-4581-8ba5-df7d05d9cd98.png)
。

**Reply by lincTHU** [reply]
上面方法小概率弾同样的错误，。不确定问题根源

**Reply by ruiye18** [reply]
FFXIVChnTextPatch-0407的版本我每次点开游戏第一次会报同样的错误，关掉大概第二次有概率能成功。。但是也比较随缘。关掉dx11也会报错

**Comment by Dzz0328** [comment]
大佬幸苦了！

**Comment by sai750310** [comment]
![圖片](https://user-images.githubusercontent.com/100848565/163103656-27ad81cf-eb66-4295-8e85-a6247d41a270.png)
使用後進入遊戲出現錯誤視窗

**Reply by Billy-MC** [reply]
我也有這問題

**Comment by corey924** [comment]
剛剛發現將簡體語系包的 rawexd-zhcn-0413\quest 目錄，覆蓋到繁體語系的 \FFXIVChnTextPatch-0413\resource\rawexd\quest 目錄後，再執行一次【漢化】就可以進去遊戲。可能是繁體語系內有什麼內容會跟 6.1 版衝突，但裡面的 csv 檔太多，不知道是哪一個有問題。另外也不確定是否其他環境也適用，漢化錯誤進不去的話可以嘗試看看。

**Reply by cydad111** [reply]
以上方式试过不行,好像汉化文本出问题.

**Reply by cydad111** [reply]
请问,因为之前忘记更新前还原汉化,不会有什么问题把?

**Reply by cydad111** [reply]
另外,我更新后进游戏是完全英文- -应该是更新时候连带着文件替换回来了.

**Reply by SmileSkys** [reply]
重新安裝後再漢化一樣會有錯誤，應該跟還原無關!  真不行就等穩定版吧!

**Comment by Csy56535** [comment]
请问该怎么备份原本的语言资料呢，以方便下次更新时可以再复制备份的挡案至ff14的语言资料夹里，变回原本的英文

**Reply by SmileSkys** [reply]
第一個檔案裡就有夾帶備份檔案了!

**Reply by Csy56535** [reply]
請問是要自己複製貼上嗎

**Reply by SmileSkys** [reply]
是的!

**Comment by ShikimiCHO** [comment]
我漢化後也開啟遊戲也失敗了，但手動還原再漢化一次就可以用了
我朋友是漢化後遊戲也爆炸，但他直接按漢化程式裡的還原，再直接進遊戲就漢化成功了(這個有點莫名其妙XD)

**Reply by Billy-MC** [reply]
我跟你說的去做, 漢化>還原>漢化 也是失敗

**Reply by cydad111** [reply]
> 我跟你說的去做, 漢化>還原>漢化 也是失敗

完全還原后,開一次漢化器,關掉再開.

**Reply by ShikimiCHO** [reply]
維修完又錯誤了，但我就重複漢化>進遊戲看看>不行就還原>再漢化進遊戲看看>還是不行我又漢化一次>還是不行我又漢化第三次>
結果就可以了我也不懂原理

**Comment by Luobiniya** [comment]
用一下午了不行，剛剛就可以漢化了

**Reply by Cutycake** [reply]
真的欸 六點回到家時漢化失敗
現在九點左右在弄一次就行了 好怪

**Reply by Billy-MC** [reply]
我到現在也還是不行...哈哈哈

**Comment by hhangchunge** [comment]
This application requires a java runtime environment16.0.1.汉化应用程序打开跳这个窗口要咋整。
之前6.08的可以打开，这次0413的就跳这个。。。。。求解。。。。。。。。。

**Comment by cydad111** [comment]
14號維護后,英語版不行了- -,用這個辦法進去還是英語

**Comment by Hyun012** [comment]
英文版dx报错不能用,日文版CSV也是报错,用国服简体的就可以

**Comment by Davidwtw** [comment]
方案一和二都試了很多次,還是不行

**Comment by earvinwu** [comment]
我用方法二，然後再把巴哈所羅門提供的6.1 0a0檔覆蓋到sqpack/ffxiv裡，是成功的，大家可以試試，我是日文版
